R/derive_merged.R

```diff
@@ -659,7 +711,7 @@ derive_vars_merged_lookup <- function(dataset,
     new_vars = new_vars,
     mode = mode,
     filter_add = !!filter_add,
-    match_flag = temp_match_flag,
+    exist_flag = temp_match_flag,
```

Comment:
@bundfussr as for using `get_new_tmp_var()` and `remove_tmp_vars()` , this is actually a wrapper for `derive_vars_merged()` so I don't think your recommendation is quite appropriate, as its already being run internally, what I should've done is not modify naming convention and just simply change the `match_flag` argument to `exist_flag` in this function, right? As for the removal in globals that is indeed gone now

Reply:
The intention of `get_new_tmp_var()` is to avoid conflicts between variables of the input dataset and temporary variables created in the function. Assume `dataset` already contains `temp_match_flag`. Then `derive_vars_merged_lookup()` would remove it. Thus I think we should use `get_new_tmp_var()` here to avoid this.